go1.13/base/Dockerfile.tmpl

```diff
@@ -23,7 +23,7 @@ RUN \
 
 ARG GOLANG_VERSION=1.13.10
 ARG GOLANG_DOWNLOAD_URL=https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
-ARG GOLANG_DOWNLOAD_SHA256=9cacc6653563771b458c13056265aa0c21b8a23ca9408278484e4efde4160618
+ARG GOLANG_DOWNLOAD_SHA256=8a4cbc9f2b95d114c38f6cbe94a45372d48c604b707db2057c787398dfbf8e7f
```

Comment:
Just for future reference: where does this sha come from?

Reply:
https://golang.org/dl/

<img width="990" alt="Screenshot 2020-07-08 at 20 20 27" src="https://user-images.githubusercontent.com/5400788/86955877-95a32000-c158-11ea-9730-9e990db21b7d.png">